### ISSUE TITLE
Steam Deck (ukmm 0.7.1) The cursor is restricted by the upper left quadrant of the screen in GameMode

### ISSUE THREAD

**Issue by Luso0** [NiceneNerd/ukmm#76]
The cursor is restricted by the upper left quadrant of the screen in GameMode. Forcing the resolution to anything does nothing.

In Desktop Mode the UI scaling is gigantic and unusable.

**Comment by USMP-lancer** [comment]
i cant even get it to run in game mode, i think the generic linux build will work better for now

**Comment by USMP-lancer** [comment]
nope, same problem, strange, probably because of the odd resolution

**Comment by NiceneNerd** [comment]
The generic Linux build usually doesn't run on Steam Deck at all because of an outdated glibc version. That's the only reason there is a Steam Deck build.

Anyway, the scaling problem is a duplicate of #18, #41, and #59.
